lib/rubocop/cop/style/space_after_not.rb

```diff
@@ -16,6 +16,8 @@ class SpaceAfterNot < Cop
         MSG = 'Do not leave space between `!` and its argument.'.freeze
 
         def on_send(node)
+          return if node.keyword_not?
```

Comment:
Is the new change okay? I avoided the double unpack, but basically replicated the `keyword_not?` method.